src/async/tasks.md

```diff
@@ -60,4 +60,10 @@ Copy this example into your prepared `src/main.rs` and run it from there.
 
 * Refactor the async block into a function, and improve the error handling using `?`.
 
+* `nc` or `telnet` are good tools for this. It's not HTTPS
```

Comment:
Let's move this line up to line 53. "..run it from there. Try connecting to it with a TCP connection tool like `nc` or `telnet`".

Reply:
i have shifted the content to line 53...
Do i need do shift the basic definitions of each `nc` and `telnet` to somewhere else as currently they just below this line